app/src/main.rs

```diff
@@ -1,20 +1,48 @@
 mod enclave_u;
 mod server;
+#[cfg(feature = "test")]
```

Comment:
Can we name this feature something else? This can easily be confused with `#[cfg(test)]`

Reply:
sure, I'll rename it to "sgx-test"